### DISCUSSION TITLE
Is it possible to preload a screen?

### DISCUSSION THREAD

**Discussion by mfbx9da4** [software-mansion/react-native-screens#1317] [Ideas]
I would like to preload a screen and keep it in memory at all times because I would like navigating to it to be super fast. Is there anyway to do that with react-native-screens? Perhaps with react-freeze?

**Comment by tboba** [comment]
Hi @mfbx9da4, thanks for creating the discussion!
I believe preloading a screen is not possible right now on react-native-screens, but it's a great idea if it's possible for now (I'm thinking about some obstacles there, for example iOS let's you to set some header attributes only when the screen finishes transitioning and has appeared).
Since it's not introduced yet, I'm moving this thread to Ideas.

Thanks!
